### PR TITLE
PDCT-1010/speed_up_tests

### PR DESCRIPTION
**Pull request by asacristani**
# Description

- Speeds up the tests (see `conftest.py`
- Standarizes the use of `with_transaction`
- Fixes bug returning incorrect slug
- Removes `physical_id` from comparison in tests (as these now change as the same db is reused)

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: app/service/collection.py [hunk 1]
```diff
@@ -107,20 +110,21 @@ def update(
     # org_id = organisation.get_id(db, collection.organisation)
```
Comment (katybaulch) [inline app/service/collection.py:110]: Commented out code

File: tests/integration_tests/conftest.py [hunk 2]
```diff
@@ -81,8 +82,46 @@ def test_db(scope="function"):
...
+
+    yield session
+
+    session.close()
+    transaction.rollback()
+
+
 @pytest.fixture
-def data_db(scope="function"):
+def data_db_slow(scope="function"):
```
Comment (katybaulch) [inline tests/integration_tests/conftest.py]: What's the rationale for leaving this old fixture in?

File: app/repository/family.py [hunk 3]
```diff
@@ -79,7 +79,7 @@ def _family_to_dto(db: Session, fam_geo_meta_org: FamilyGeoMetaOrg) -> FamilyRea
         category=str(f.family_category),
         status=str(f.family_status),
         metadata=metadata,
-        slug=str(f.slugs[0].name if len(f.slugs) > 0 else ""),
+        slug=str(f.slugs[-1].name if len(f.slugs) > 0 else ""),
```
Comment (katybaulch) [inline app/repository/family.py:82]: Nice driveby!

**Review by katybaulch — CHANGES_REQUESTED** [review]
Looks great - thankyou so much for this, it's such an important change 🙏🏻 Couple of quick ones but all good other than that

File: tests/integration_tests/conftest.py (hunk 2 above)
Comment (diversemix) [reply tests/integration_tests/conftest.py]: old commit

File: app/service/collection.py (hunk 1 above)
Comment (diversemix) [reply app/service/collection.py:110]: See todo

**Review by diversemix — APPROVED** [review]

**Review by katybaulch — APPROVED** [review]
Lgtm
